conformance-suites/2.0.0/conformance2/textures/misc/copy-texture-image.html

```diff
@@ -80,8 +80,12 @@
         "RG8",
         "RGB8",
         "RGBA8",
+        "R8_SNORM",
```

Comment:
Yes, thanks @shannonwoods for finding this!

To summarize for people who don't have access to the bug: SNORM formats aren't supported in CopyTexImage2D.